_specifications/lsp/3.17/types/position.md

```diff
@@ -10,13 +10,51 @@ interface Position {
 	line: uinteger;
 
 	/**
-	 * Character offset on a line in a document (zero-based). Assuming that
-	 * the line is represented as a string, the `character` value represents
-	 * the gap between the `character` and `character + 1`.
+	 * Character offset on a line in a document (zero-based). The meaning of this
+	 * offset is determined by the negotiated `PositionEncodingKind`.
 	 *
 	 * If the character value is greater than the line length it defaults back
 	 * to the line length.
 	 */
 	character: uinteger;
 }
-```
\ No newline at end of file
+```
+
+When describing positions the protocol needs to specify how offsets (specifically character offsets) should be interpreted.
+The corresponding `PostionEncodingKind` is negotiated between the server and client and server during initialization.
```

Comment:
I think this should read ... is negotiated between the server and the client during initialization.

Reply:
Yes, thanks.

Reply:
Fixed.